doc/source/redis-memory-management.rst

```diff
@@ -0,0 +1,103 @@
+Redis Memory Management (EXPERIMENTAL)
+==============================================
+
+Ray stores metadata associated to tasks and objects in one or more Redis
+servers, as described in :doc:`internals-overview`.  Applications that are
+long-running or have high task/object generation rate could risk high memory
+pressure, potentially leading to out-of-memory (OOM) errors.
+
+Here, we describe an experimental feature that transparently flushes metadata
+entries out of Redis memory.
+
+Requirements
+-------------
+
+As of early July 2018, the automatic memory management feature requires building
+Ray from source.  We are planning on eliminating this step in the near future by
+releasing official wheels.
+
+Building Ray
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, follow `instructions to build Ray from source
+<installation.html#building-ray-from-source>`__ to install prerequisites.  After
```

Comment:
This link doesn't seem to work. Maybe just use an absolute path to the URL.

Reply:
See my other comment?

Reply:
Ok, I see this pattern in the tune documentation.